.gitignore

```diff
@@ -1,5 +1,6 @@
 # Project specific files
 /hooks/settings_local.py
+/hooks/settings/_settings_local.py
```

Comment:
@jimrkh Maybe add a comment that the old location can be removed when this is deployed on master :-)